### ISSUE TITLE
Biome IDs not changing when using Config option

### ISSUE THREAD

**Issue by Kolrolf** [micdoodle8/Galacticraft#2059]
I'm trying to change the biome IDs for Galacticraft in core.conf using I:biomeIDBase but the biome IDs are not changing.  Setting I:biomeIDBase=103; saving the config file; creating a new world and using NEI to dump the biome IDs I get the following:
102	moon
103	marsFlat
104	null
105	asteroids
106	null

The asteriods biome ID has adjusted acording to the config, but moon and mars did not, and space biome doesn't show up.

I've tried this with Galacticraft 3.0.12.375 and Galacticraft 3.0.12.437, same result.
Using Forge version 13.4.1566

I have the following mods installed:
![mod-list-1](https://cloud.githubusercontent.com/assets/17055792/12801133/311be5b4-caa8-11e5-941c-80a7bed93e3c.png)
![mod-list-2](https://cloud.githubusercontent.com/assets/17055792/12801134/32c14c6a-caa8-11e5-8665-02e599cfd15f.png)

**Comment by radfast** [comment]
I'm not seeing this.  Please post extract from your biome.csv.

**Comment by Kolrolf** [comment]
No problem, had to convert it to xlsx to upload:

[biome.xlsx](https://github.com/micdoodle8/Galacticraft/files/116734/biome.xlsx)

**Comment by radfast** [comment]
I'm getting this:

> 103,moon,0.5,0.0,0.1,1.5,0.4,"COLD, DRY, DEAD",micdoodle8.mods.galacticraft.core.world.gen.BiomeGenFlatMoon
104,marsFlat,0.5,0.0,0.01,2.5,0.4,"COLD, DRY, DEAD, SANDY",micdoodle8.mods.galacticraft.planets.mars.world.gen.BiomeGenFlatMars
105,asteroids,0.5,0.0,0.01,0.1,0.2,"COLD, DRY, SPOOKY, DEAD",micdoodle8.mods.galacticraft.planets.asteroids.world.gen.BiomeGenBaseAsteroids
106,space,0.5,0.0,0.01,0.1,0.2,"DRY, PLAINS",micdoodle8.mods.galacticraft.core.world.gen.BiomeGenBaseOrbit

It should be working for you too.  The only thing I can think of is that one of the other mods is causing this to operate differently - maybe if you try using a different biomeIDbase in a space where there are 4 null biome IDs currently?

**Comment by Kolrolf** [comment]
Just tried setting biomeIDbase to 201, new world, and got:

> 
102	moon	0.5	0	0.1	1.5	0.4	COLD, DRY, SPOOKY, DEAD, NETHER	micdoodle8.mods.galacticraft.core.world.gen.BiomeGenFlatMoon
103	marsFlat	0.5	0	0.01	2.5	0.4	COLD, DRY, DEAD, SANDY	micdoodle8.mods.galacticraft.planets.mars.world.gen.BiomeGenFlatMars
104	null	null	null	null
105	space	0.5	0	0.01	0.1	0.2	DRY, PLAINS	micdoodle8.mods.galacticraft.core.world.gen.BiomeGenBaseOrbit
...
203	asteroids	0.5	0	0.01	0.1	0.2	COLD, DRY, SPOOKY, DEAD	micdoodle8.mods.galacticraft.planets.asteroids.world.gen.BiomeGenBaseAsteroids

I'll try removing mods to see which one's the cause.

**Comment by radfast** [comment]
Thank you, that will be great.
I should also ask this - are you single player or on a server?  If on a server, do server and client configs match?

**Comment by Kolrolf** [comment]
It's single player

After removing mods and adding them back in I've determined that it's BiomeTweaker that's causing the issue.  I've checked and it's not the script I wrote (which only removed mobs from a few biomes anyways), it seems like the issue is BiomeTweaker affecting Galacticraft biomes in some odd way.

Thanks for your support with my issue and quick responses, I'll take this over to the BiomeTweaker issue tracker.
